pom.xml

```diff
@@ -89,6 +89,7 @@
         <confluent-log4j.version>1.2.17-cp2</confluent-log4j.version>
         <zkclient.version>0.11</zkclient.version>
         <zookeeper.version>3.5.9</zookeeper.version>
+        <netty.version>4.1.61</netty.version>
```

Comment:
Do we want .61 or .63? I'm not sure how netty versioning works so not sure if those are equally safe... although I suppose this does seem like the minimal update required.

Reply:
@ewencp changed the version to 4.1.63 since that is the version schema registry is already using.

Reply:
Sounds good, still lgtm